pages/cart.tsx

```diff
@@ -111,7 +110,9 @@ const CartView : NextPage = () => {
           </CartItems>
         )}
 
-        <p style={{ padding: "1rem", margin: "1rem 0", backgroundColor: `${tokens.colors.ui.background__warning.rgba}` }}>Placeholder for a banner component</p>
+        <Banner variant="warning">
+          {intl.formatMessage({ id: "cart.banner.warning" })}
```

Comment:
We need to wrap this in the Typography component to get the font correct and also to get a `<p>` tag